sql/memo/exec_builder.go

```diff
@@ -56,6 +56,26 @@ func (b *ExecBuilder) buildFullOuterJoin(j *FullOuterJoin, input sql.Schema, chi
 	return plan.NewFullOuterJoin(children[0], children[1], filters), nil
 }
 
+func (b *ExecBuilder) buildLateralCrossJoin(j *LateralCrossJoin, input sql.Schema, children ...sql.Node) (sql.Node, error) {
+	return plan.NewLateralCrossJoin(children[0], children[1]), nil
+}
+
+func (b *ExecBuilder) buildLateralInnerJoin(j *LateralInnerJoin, input sql.Schema, children ...sql.Node) (sql.Node, error) {
+	filters, err := b.buildFilterConjunction(j.g.m.scope, input, j.Filter...)
```

Comment:
wondering if we'd ever need to reindex this entire node, depending on it if is commuted in the larger join tree